protocol/contracts/dao/Comptroller.sol

```diff
@@ -123,7 +123,9 @@ contract Comptroller is Setters {
     }
 
     function mintToPool(uint256 amount) private {
-        if (amount > 0) {
+        if (amount > 0 && _hybridLiquidityState.isEnabled) {
+            hybridOraclePool().distributeToPools(amount);
+        } else if (amount > 0) {
             gold().mint(pool(), amount);
         }
     }
```

Comment:
As there are now multiple pools encapsulated within the `hybridOraclePool`, it's better to defer the mint to it.
The other option would be returning the list of pools here, but it seemed a bit messy. Note that this requires the new HybridOraclePool to be added to the list of minters for ESG, and this is done as part of the Implementation `initialize` during commit.